app/controllers/internal/page_redirects_controller.rb

```diff
@@ -0,0 +1,62 @@
+class Internal::PageRedirectsController < Internal::ApplicationController
```

Comment:
should we add auditing support here? 

See https://github.com/thepracticaldev/dev.to/pull/6797/files and https://github.com/thepracticaldev/dev.to/blob/master/app/controllers/internal/articles_controller.rb

Reply:
I didn't even know this was a thing 🙈 . At a glance, it seems like that makes sense. I'll have to look at the documentation and see what that's all about 💯.

Reply:
Added 👍